packages/framework/src/Resources/config/services.yml

```diff
@@ -720,7 +720,7 @@ services:
         alias: Shopsys\FrameworkBundle\Model\Order\OrderNumberSequenceFactory
 
     Shopsys\FrameworkBundle\Model\Order\Item\OrderItemFactoryInterface:
-        alias: Shopsys\FrameworkBundle\Model\Order\Item\OrderItemFactory
+        alias: Shopsys\ShopBundle\Model\Order\Item\OrderItemFactory
```

Comment:
We do not use ShopBundle classes in framework, because framework package does not depend on project-base